src/vunnel/providers/nvd/manager.py

```diff
@@ -55,36 +58,44 @@ def get(
         last_updated: datetime.datetime | None,
         skip_if_exists: bool = False,
     ) -> Generator[tuple[str, dict[str, Any]], Any, None]:
-        self.overrides.download()
+        """main method to fetch and process NVD vulnerability data.
```

Comment:
Thanks for all the really clear doc comments in here. It made reviewing this class much easier.

Reply:
it tripped me up, so figured it needed a sprinkling of comments ✏️